asr/review.py

```diff
@@ -156,6 +150,13 @@ def review(dataset,
                 1: 1 / frac_included
             }
 
+        # create the model
+        model = KerasClassifier(
+            create_lstm_model(embedding_matrix=embedding_matrix,
+                              verbose=verbose),
+            **fit_kwargs
```

Comment:
`fit_kwargs` are meant for fitting. Does this one belong here?

Reply:
It seems so yes. That's how I got the number of epochs to work.